src/internal/connector/exchange/exchange_data_collection.go

```diff
@@ -266,6 +267,7 @@ func (col *Collection) streamItems(ctx context.Context, errs *fault.Bus) {
 			}
 
 			info.Size = int64(len(data))
+			info.Path = strings.Join(col.fullPath.Folders(), "/")
```

Comment:
path package has a join func that you should use here.

Reply:
https://github.com/alcionai/corso/blob/main/src/pkg/path/path.go#L590 is not a better use case for the above. If we decide the application is standard enough to create a helper function, we should include it in a follow-up PR.